modules/reindex/src/test/java/org/opensearch/index/reindex/CancelTests.java

```diff
@@ -116,7 +116,7 @@ private void testCancel(
             false,
             true,
             IntStream.range(0, numDocs)
-                .mapToObj(i -> client().prepareIndex(INDEX, TYPE, String.valueOf(i)).setSource("n", i))
+                .mapToObj(i -> client().prepareIndex(INDEX, MapperService.SINGLE_MAPPING_NAME, String.valueOf(i)).setSource("n", i))
```

Comment:
Should we remove the type completely here as we are removing _doc type info from [validation ymls](https://github.com/opensearch-project/OpenSearch/pull/2205/files#diff-bab5c3316c52f94500525d112ab9e6d6f71c792286ca32c24fb744294a20b8f2)

Reply:
Type is still needed here for cancellation tests because it hasn't yet been fully removed from the DocWriter. That'll be in a coming PR